### DISCUSSION TITLE
ScriptReference/MonoBehaviour.Invoke

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#367] [Notes]
# ScriptReference/MonoBehaviour.Invoke

https://docs.unity3d.com/ScriptReference/MonoBehaviour.Invoke.html

**Comment by docscomments** [comment]
**Comment by `@Juskelis`:**  

Note that Invoke follows scaled time. If you need to Invoke regardless of timescale, you'll need to write your own invoker.

**Comment by vertxxyz** [comment]
If you're invoking a public method, use [`nameof`](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/nameof) to get a string that will be maintained through refactoring.
